prac_06/guitars.py

```diff
@@ -0,0 +1,29 @@
+"""
+Estimated Time: 15 Minutes
+Actual Time: 38 Minutes
+"""
+
+from prac_06.guitar import Guitar
+
+
+def main():
+    print("My guitars!")
+    guitars = []
+    guitar_number = 1
+
+    name = input("Name: ")
+    year = int(input("Year: "))
+    cost = float(input("Cost: "))
```

Comment:
I believe this is meant to loop until a blank name is entered
name = 
e.g. while name != ""
year
cost